pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -154,7 +154,7 @@ func (c *contract) Reveal(ctx context.Context, storageDepth uint8, reserveCommit
 		Value:       big.NewInt(0),
 		Description: "reveal transaction",
 	}
-	err = c.sendAndWaitWithBoost(ctx, request, 50)
+	err = c.sendAndWait(ctx, request, 50)
```

Comment:
I've been taking a slightly less aggressive approach with Reveal only boosting 10-20%, commit boosting 50%, and claim boosting 25-50%.   If a node misses the reveal, nothing is really lost but the chance to play.   Missing the commit is really bad.   Missing the claim is not as bad, but can also cost a LOT more gas.   Until the EIP transactions are put in place, this boost is a guaranteed increase in the gas paid for the transactions.

Reply:
We have work in progress to move to the new EIP transactions.
We are also adding new mechanisms to prevent the overloaded postage contract expire() from causing transactions to run of gas, which should lead to reductions in gas costs everywhere.